recipe/meta.yaml

```
@@ -25,20 +28,42 @@ requirements:
     - python
     - py4j ==0.10.9.7
     # extras_require dependencies
-    - numpy >=1.15
+    - numpy >=1.15,<2
     - pandas >=1.0.5
-    - pyarrow >=1.0.0
+    - pyarrow >=4.0.0
+    - grpcio >=1.56.0
+    - grpcio-status >=1.56.0
+    - googleapis-common-protos >=1.56.4
```

```suggestion
  run_constrained:
    - grpcio >=1.56.0
    - grpcio-status >=1.56.0
    - googleapis-common-protos >=1.56.4
```
We haven't previously included the "connect" extras, so we can keep these in run_constrained.